libs/cardano-ledger-core/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.1.0.0
 
+* Rename `UMapCompact` to `UMap` and add tests for it.
```

Comment:
`1.1.0.0` has now been released, so this PR should start a `1.2.0.0`. This will require a bump in cabal file as well.

You should record that the module name has changed. The way it is written now looks like a type rename. 
```suggestion
* Rename `Cardano.Ledger.UMapCompact` to `Cardano.Ledger.UMap` and add tests for it.
```
Also there is no mention of new function: `rdPairView`

Moreover, every package that has been affected by this rename, must have its lower bound be updated.